docs/Configuration.md

```diff
@@ -32,6 +32,60 @@ SQLite            | `SQLite`
 
 TBD
 
+## LANGUAGE section
+
+The LANGUAGE section has one key, `locale`.
+
+The value of the `locale` key is a space separated list of
+`locale tags` where each tag must match the regular expression
+`/^[a-z]{2}_[A-Z]{2}$/`.
+
+The two first characters of a `locale tag` are intended to be an
+[ISO 639-1] two-character language code and the two last characters
+are intended to be an [ISO 3166-1 alpha-2] two-character country code.
+A `locale tag` is a locale setting for the available translation
+of messages without ".UTF-8", which is implied.
+
+Adding a new `locale tag` to the configuration requires that the
+equivalent .mo file is added to Zonemaster-Engine at the correct
+place so that gettext get retrieve it. See the
+[Zonemaster-Engine share directory] for the existing .po files
+that are converted to .mo files. (Here we should have a link
+to documentation instead.)
+
+Removing a language from the configuration file just blocks that
+language from being allowed. If there are more than one `locale tag`
+(with different country codes) for the same language, then
+all those must be removed to block that language.
+
+English is the Zonemaster default language, but it can be blocked
+from being allowed by RPC-API by not including it in the
+configuration.
+
+The default installation and configuration supports the
+following languages.
+
+Language | Locale tag value   | Locale value used
+---------|--------------------|------------------
+Danish   | da_DK              | da_DK.UTF-8
+English  | en_US              | en_US.UTF-8
+French   | fr_FR              | fr_FR.UTF-8
+Swedish  | sv_SE              | sv_SE.UTF-8
+
+It is an error to repeate the same `locale tag`.
+
+Setting in the default configuration file:
+
+```
+locale = da_DK en_US fr_FR sv_SE
+```
+
+If the `locale` key is empty or absent, the `locale tag` value
+"en_US" is set by default.
```

Comment:
This should be moved up and placed together with the other notes on the semantics of the locale property.

Reply:
I do not see the problem, but I moved it up.